homeassistant/components/trafikverket_weatherstation/sensor.py

```diff
@@ -243,5 +227,7 @@ async def async_update(self) -> None:
         except (asyncio.TimeoutError, aiohttp.ClientError, ValueError) as error:
             _LOGGER.error("Could not fetch weather data: %s", error)
             return
-        self._active = self._weather.active
-        self._measure_time = self._weather.measure_time
+        self._attr_extra_state_attributes = {
```

Comment:
You should probably split out self._attr_native_value out of the try...except
I assume the first two exceptions only apply to the api call.
If needed wrap it in a separate try...except for the ValueError